generators/config/templates/drivers/headless_firefox.rb

```diff
@@ -0,0 +1,15 @@
+# :headless_firefox driver
+
+Capybara.register_driver :headless_firefox do |app|
+  startup_flags = ['--headless']
+  startup_flags.concat(Howitzer.headless_firefox_flags.split(/\s*,\s*/)) if Howitzer.headless_firefox_flags
```

Comment:
Please add custom user agent support, like we do for other drivers. For example, https://github.com/strongqa/howitzer/blob/e1995fbb0a66b2bc6dc4c71cf82a59b76ecbe28e/generators/config/templates/drivers/headless_chrome.rb#L6